### PR TITLE
restructure install topics for Replication Server

### PR DESCRIPTION
**Pull request by dwicinas**
Restructure and in places rewrite all information for Linux installations of Replication Server

### CODE REVIEW

**Comment by dwicinas** [comment]
@drothery-edb, here's a first pass at restructuring Replication Server. I moved lots of information around. In places it feels pretty rough, but maybe it's good enough for this interim stage. (Note that I squashed all commits for this PR into one.)

File: product_docs/docs/eprs/7/02_overview/02_replication_concepts_and_definitions/10_sync_replication_log_based/04_replication_origin.mdx [hunk 1]
```diff
@@ -58,7 +58,7 @@ The following example shows some of the replication slot information for a three
 
 The following shows the maximum allowable number of replication slots:
 
-```text
+```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/02_overview/02_replication_concepts_and_definitions/10_sync_replication_log_based/04_replication_origin.mdx:61]: This should be `sql`

File: product_docs/docs/eprs/7/02_overview/02_replication_concepts_and_definitions/13_table_filters.mdx [hunk 2]
```diff
@@ -41,25 +41,25 @@ id NUMERIC(2) PRIMARY KEY
 
 You can use the following filters:
 
-```text
+```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/02_overview/02_replication_concepts_and_definitions/13_table_filters.mdx:44]: Should be `sql`

File: product_docs/docs/eprs/7/02_overview/02_replication_concepts_and_definitions/13_table_filters.mdx [hunk 3]
```diff
@@ -41,25 +41,25 @@ id NUMERIC(2) PRIMARY KEY
...
 deptno IN (SELECT id FROM public.dept_filter_lookup)
 ```
 or 
 
-```text
+```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/02_overview/02_replication_concepts_and_definitions/13_table_filters.mdx:49]: Should be `sql`

File: product_docs/docs/eprs/7/02_overview/03_replication_server_components_and_architecture/01_physical_components.mdx [hunk 4]
```diff
@@ -105,7 +105,7 @@ Replication Server creates the content of this file as follows:
 
 The following is an example of the content of an EPRS Replication Configuration file:
 
-```text
+```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/02_overview/03_replication_server_components_and_architecture/01_physical_components.mdx:108]: Not shell; try `ini`

File: product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx [hunk 5]
```diff
@@ -12,7 +12,7 @@ Keep the following points in mind when designing a replication system:
 -   Each non-PostgreSQL/EDB Postgres Advanced Server database table used in a publication must have a primary key, with the exception of tables in snapshot-only publications. These tables don't require a primary key. 
 -   Each PostgreSQL/EDB Postgres Advanced Server database table used in a publication must have identity columns (primary key or unique columns), with the exception of tables in snapshot-only publications. These tables don't require identity columns. For example:
 
-```text
+```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx:15]: Should be `sql`

File: product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx [hunk 6]
```diff
@@ -22,7 +22,7 @@ CREATE TABLE dept_pk (
 
 If a table doesn't have a primary key, then it should have a column with `UNIQUE NOT NULL` constraints. For example:
 
-```text
+```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/02_overview/04_design_replication_system/02_design_considerations.mdx:25]: Should be `sql`

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+    In the directory `/etc/yum.repos.d`, the repository configuration file `edb.repo` is created, which a text file is containing a list of EnterpriseDB repositories, each denoted by an entry starting with the text `[repository_name]`.
+
+2. Enable access to the packages in any of these repositories:
+
+    - Using your requested credentials for the EDB Yum Repository, substitute the user name and password for the `<username>:<password>` placeholders of the baseurl parameter.
+    - Change the setting of the enabled parameter to `enabled=1`.
+  
+  For example, to access the server `libs` package from the repository for Advanced Server version 14, enable the following entry:
+
+  ```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx:38]: Try `ini`

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+  enabled=1
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/ENTERPRISEDB-GPG-KEY
+  ```
+
+  When enabling one version, be sure the other entries are disabled (that is, the parameter setting is `enabled=0` for the non-selected entries). 
+    
+  For example, to access Replication Server version 7.0, enable the following entry:
+
+  ```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx:51]: ini

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+
+| Package name         | Component                                                                  |
+| -------------------- | -------------------------------------------------------------------------- |
+| `edb-xdb-console`    | EPRS Replication Console and the Replication Server Command Line Interface |
+| `edb-xdb-publisher`  | Publication server                                                         |
+| `edb-xdb-subscriber` | Subscription server                                                        |
+
+The following is an example:
+
+```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx:86]: FWIW: you can separate the output from the command by including `__OUTPUT__` on a line by itself

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/02_eprs_other_linux8_x86.mdx [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+    In the directory `/etc/yum.repos.d`, the repository configuration file `edb.repo` is created, which a text file is containing a list of EnterpriseDB repositories, each denoted by an entry starting with the text `[repository_name]`.
+
+2. Enable access to the packages in any of these repositories:
+
+    -   Using your requested credentials for the EDB Yum Repository, substitute the user name and password for the `<username>:<password>` placeholders of the baseurl parameter.
+    -   Change the setting of the enabled parameter to `enabled=1`.
+
+  For example, to access the server `libs` package from the repository for Advanced Server version 14, enable the following entry:
+
+  ```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/02_eprs_other_linux8_x86.mdx:38]: ini

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/02_eprs_other_linux8_x86.mdx [hunk 11]
```diff
@@ -0,0 +1,213 @@
...
+  enabled=1
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/ENTERPRISEDB-GPG-KEY
+  ```
+
+  When enabling one version, be sure the other entries are disabled (that is, the parameter setting is `enabled=0` for the non-selected entries). 
+    
+  For example, to access Replication Server version 7.0, enable the following entry:
+
+  ```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/02_eprs_other_linux8_x86.mdx:51]: ini

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/03_eprs_rhel7_x86.mdx [hunk 12]
```diff
@@ -0,0 +1,213 @@
...
+    In the directory `/etc/yum.repos.d`, the repository configuration file `edb.repo` is created, which a text file is containing a list of EnterpriseDB repositories, each denoted by an entry starting with the text `[repository_name]`.
+
+2. Enable access to the packages in any of these repositories:
+
+    -   Using your requested credentials for the EDB Yum Repository, substitute the user name and password for the `<username>:<password>` placeholders of the baseurl parameter.
+    -   Change the setting of the enabled parameter to `enabled=1`.
+
+  For example, to access the server `libs` package from the repository for Advanced Server version 14, enable the following entry:
+
+  ```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/03_eprs_rhel7_x86.mdx:38]: ini

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/04_eprs_centos7_x86.mdx [hunk 13]
```diff
@@ -0,0 +1,216 @@
...
+      You may have to enable the \[extras] repository definition in the `CentOS-Base.repo file (located in /etc/yum.repos.d)`.
+
+2. Enable access to the packages in any of these repositories:
+
+    -   Using your requested credentials for the EDB Yum Repository, substitute the user name and password for the `<username>:<password>` placeholders of the baseurl parameter.
+    -   Change the setting of the enabled parameter to `enabled=1`.
+
+  For example, to access the server `libs` package from the repository for Advanced Server version 14, enable the following entry:
+
+  ```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/04_eprs_centos7_x86.mdx:41]: ini

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/04_eprs_centos7_x86.mdx [hunk 14]
```diff
@@ -0,0 +1,216 @@
...
+  enabled=1
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/ENTERPRISEDB-GPG-KEY
+  ```
+
+  When enabling one version, be sure the other entries are disabled (that is, the parameter setting is `enabled=0` for the non-selected entries). 
+    
+  For example, to access Replication Server version 7.0, enable the following entry:
+
+  ```shell
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/04_eprs_centos7_x86.mdx:54]: ini

**Review by josh-heyer — CHANGES_REQUESTED** [review]
Didn't go through all of these, but called out some representative examples of code blocks with incorrect language tags. Please try to get as close as possible to the type of text being rendered - will greatly improve syntax highlights

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/04_eprs_centos7_x86.mdx (hunk 14 above)
Comment (drothery-edb) [reply product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/04_eprs_centos7_x86.mdx:54]: Thanks for your comments, @josh-heyer. I'm going to have @dwicinas take a look at this next time he is in. We'll probably try and fix this in the install topics (the scope of the changes for this PR) and make a ticket for our backlog to review the text blocks in the rest of the doc.

File: product_docs/docs/mongo_data_adapter/5/04_installing_the_mongo_data_adapter/x86_amd64/01_mongo_rhel8_x86.mdx [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Installing MongoDB Foreign Data Wrapper on RHEL 8/OL 8 x86"
+title: "Installing Replication Server on RHEL 8/OL 8 x86"
```
Comment (josh-heyer) [inline product_docs/docs/mongo_data_adapter/5/04_installing_the_mongo_data_adapter/x86_amd64/01_mongo_rhel8_x86.mdx:2]: Missed this on the first pass - this is definitely still the MongoFDW install instructions, not EPRS - best revert :)

File: product_docs/docs/eprs/7/03_installation/01_installing_with_stackbuilder.mdx [hunk 16]
```diff
@@ -1,195 +0,0 @@
----
```
Comment (josh-heyer) [inline product_docs/docs/eprs/7/03_installation/01_installing_with_stackbuilder.mdx:1]: With this deleted, links to /eprs/latest/03_installation/01_installing_with_stackbuilder/ fail - which includes the links in /eprs/latest/05_smr_operation/01_prerequisites/06_verifying_host_accessibility/
/eprs/latest/05_smr_operation/02_creating_publication/01_registering_publication_server/
/eprs/latest/05_smr_operation/03_creating_subscription/01_registering_subscription_server/
/eprs/latest/10_appendix/02_upgrading_to_xdb6_2/02_upgrading_with_gui_installer/

File: product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx (hunk 7 above)
Comment (drothery-edb) [reply product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx:38]: I think we might be better off with text. I assume ini should color code the values to the right of the equals sign, but it is just color coding the numbers and not the rest of the text.

**Pull request merged**